WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -123,8 +130,10 @@ public class BlockEditorScreen: ScreenObject {
     }
 
     private func addBlock(_ blockLabel: String) {
+        let blockButton = XCUIApplication().buttons[blockLabel]
```

Comment:
```suggestion
        let blockButton = app.buttons[blockLabel]
```

`ScreenObject` as an [`app`](https://github.com/Automattic/ScreenObject/blob/2d65beae47cdcaaf21703ce1b321f382fe0d0730/Sources/ScreenObject/ScreenObject.swift#L21-L23) property which exposes its [required `XCUIApplication` init parameter](https://github.com/Automattic/ScreenObject/blob/2d65beae47cdcaaf21703ce1b321f382fe0d0730/Sources/ScreenObject/ScreenObject.swift#L41-L45). 

99% of the time, `app` and `XCUIApplication()` are going to be the same. The real advantage is in not having to re-instantiate the object.